src/FunctionApps/python/tests/IntakePipeline/assets/FileSingleMessageLongTZ.hl7

```diff
@@ -1,10 +1,6 @@
-FHS|^~\&|WIR11.3.2|WIR|||20200514||1219144.update|||
-BHS|^~\&|WIR11.3.2|WIR|||20200514|||||
-MSH|^~\&|WIR11.3.2^^|WIR^^||WIRPH^^|20200514||VXU^V04|2020051411020600|P^|2.4^^|||ER
-PID|||3054790^^^^SR^~^^^^PI^||ZTEST^PEDIARIX^^^^^^|HEPB^DTAP^^^^^^|2018080812345235|M|||||||||||||||||||||
```

Comment:
How come we're deleting lines out of this test case? If it's supposed to be a bad test I would think we still want the parser robust against whatever these lines and the below ones were testing.

Reply:
I had uploaded this batch message with another ticket, but realized there weren't actually tests written around it, so I renamed and modified it to fit the testing I wanted to do for this.